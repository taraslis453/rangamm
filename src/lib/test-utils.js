import React from 'react'
import {render} from '@testing-library/react'
import {theme} from '../theme'
import {ThemeProvider} from 'styled-components'
import {BrowserRouter} from 'react-router-dom'
import {store} from '../store'
import {Provider} from 'react-redux'
const AllTheProviders = ({children}) => {
  return (
    <div id='root'>
      <BrowserRouter>
        <Provider store={store}>
          <ThemeProvider theme={theme}>{children}</ThemeProvider>
        </Provider>
      </BrowserRouter>
    </div>
  )
}

const customRender = (ui, options) => render(ui, {wrapper: AllTheProviders, ...options})
export * from '@testing-library/react'

export {customRender as render}
