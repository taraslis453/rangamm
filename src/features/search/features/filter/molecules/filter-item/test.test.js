import React from 'react'
import {render} from 'lib/test-utils'
import {FilterItem} from './index'
describe('<FilterItem/>', () => {
  it('renders properly', () => {
    const {container} = render(
      <FilterItem title='Orientation'>
        <div>Child</div>
      </FilterItem>,
    )
    expect(container.firstChild).toMatchSnapshot()
  })
})
