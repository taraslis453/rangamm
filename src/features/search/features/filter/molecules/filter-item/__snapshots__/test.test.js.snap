// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilterItem/> renders properly 1`] = `
<div
  id="root"
>
  <div
    class="sc-hKgILt bSNoIA"
    data-testid="filterItem"
  >
    <div
      class="sc-hKgILt dsStyI"
    >
      <div>
        Orientation
      </div>
      <div>
        Child
      </div>
    </div>
  </div>
</div>
`;
